packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -52,11 +52,16 @@
 	}
 }
 
-.edit-site-global-styles-header__description,
+.edit-site-global-styles-header__description {
+	padding: 0 $grid-unit-20;
+}
+
 .edit-site-block-types-search {
+	margin-bottom: $grid-unit-10;
 	padding: 0 $grid-unit-20;
 }
 
+
```

Comment:
Spacing nit: I don't think we need this empty line?

Reply:
Agreed! That was unintentional but is fixed now. :) Thank you!